fix: 修复无法录入人脸数据的问题

KT-lcz opened the pull request:
由于系统删除了之前的预装包libqt5multimedia5-plugins，导致QCameraInfo::availableCameras()拿不到可用摄像头，所以人脸服务无法打开摄像头导致

添加libqt5multimedia5-plugins依赖保证功能可用

Log: 添加libqt5multimedia5-plugins依赖保证设备信息获取正常
Influence: 人脸服务录入等功能正常
Bug: https://pms.uniontech.com/bug-view-141827.html
Change-Id: Icd2bd46c565a4e781fbf0ddc0a95c0e123caa2a3

BLumia commented:
```patch
From 204d6d79da98353fffd8f2fbf095cf8c879d94a6 Mon Sep 17 00:00:00 2001
From: KT-lcz <54447789+KT-lcz@users.noreply.github.com>
Date: Mon, 17 Oct 2022 15:34:22 +0800
Subject: [PATCH] =?UTF-8?q?fix:=20=E4=BF=AE=E5=A4=8D=E6=97=A0=E6=B3=95?=
 =?UTF-8?q?=E5=BD=95=E5=85=A5=E4=BA=BA=E8=84=B8=E6=95=B0=E6=8D=AE=E7=9A=84?=
 =?UTF-8?q?=E9=97=AE=E9=A2=98?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
```

(https://github.com/linuxdeepin/deepin-face/commit/204d6d79da98353fffd8f2fbf095cf8c879d94a6.patch)

如果一定要在线编辑的话，建议把公司邮箱添加到 GitHub 账号中，这样在进行在线提交时可以选择使用哪个邮箱。

Commit:
fix: 修复无法录入人脸数据的问题

由于系统删除了之前的预装包libqt5multimedia5-plugins，导致QCameraInfo::availableCameras()拿不到可用摄像头，所以人脸服务无法打开摄像头导致

添加libqt5multimedia5-plugins依赖保证功能可用

Log: 添加libqt5multimedia5-plugins依赖保证设备信息获取正常
Influence: 人脸服务录入等功能正常
Bug: https://pms.uniontech.com/bug-view-141827.html
